src/components/Settings/ChangeEmail.vue

```diff
@@ -51,6 +52,15 @@ import VerificationWarning from '@/components/Settings/VerificationWarning'
 export default {
   components: { QField, QInput, QBtn, VerificationWarning },
   mixins: [statusMixin],
+  computed: {
+    hasEmailChanged () {
+      const previousEmail = this.user.email || this.user.unverifiedEmail
```

Comment:
I think `user.email` is always set, isn't it? Then you don't need to look up `user.unverifiedEmail`.

Reply:
@tiltec i will check it and do the necessary changes :+1:

Reply:
@tiltec  ya user.email works fine :) updated the changes thank you.

Reply:
No, it has to be 
```code
const previousEmail = this.user.unverifiedEmail || this.user.email
```
because otherwise it's not possible to restore one's previous email address before verifying the new one.

Reply:
Sorry didn't saw that will change it now.